imports/plugins/included/payments-stripe-connect/server/i18n/el.json

```diff
@@ -0,0 +1,24 @@
+[{
+  "language": "Greek",
+  "i18n": "el",
+  "ns": "reaction-stripe-connect",
+  "translation": {
+    "reaction-payments": {
+      "admin": {
+        "shortcut": {
+          "stripeConnectLabel": "Ταινία"
```

Comment:
The i18n files currently match what's in payments-stripe. I can change stripeConnectLabel and stripeConnectSettingsLabel as you suggested but wanted to double check. Also, should I make that change for stripeLabel and stripeSettingsLabel?